File: DTOs/Sources/MeetingServiceDTOs/PatchVotingDTO.swift
import Foundation

public struct PatchVotingDTO: Codable {
    public var question: String?
    public var description: String?
    public var anonymous: Bool?
    public var options: [GetVotingOptionDTO]? // If options is not nil: ALL previous options will be deleted/replaced
    
    public init(question: String? = nil, description: String? = nil, anonymous: Bool? = nil, options: [GetVotingOptionDTO]? = nil) {
        self.question = question
        self.description = description
        self.anonymous = anonymous
        self.options = options
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/MeetingServiceDTOs/PatchVotingDTO.kt
package net.ipv64.kivop.dtos.MeetingServiceDTOs

import java.util.UUID

import java.time.LocalDateTime

data class PatchVotingDTO (
    var question : String?,
    var description : String?,
    var anonymous : Boolean?,
    var options : List<GetVotingOptionDTO>?, // If options is not nil: ALL previous options will be deleted/replaced
)
